### PR TITLE
Fix `gap` property related issue

### PR DESCRIPTION
**Pull request by IsAmrish**
Signed-off-by: isamrish <askmaurya48@gmail.com>

This commit will fix the CSS property gap related issue in safari. When `gap` is being used with `display: flex`, it was causing issue, especially in safari browser.

fixes: https://github.com/openebs/website/issues/133

Some screenshots (fixed ones):

Screenshot 2021-06-28 at 1 20 01 PM

Screenshot 2021-06-28 at 2 30 09 PM

Screenshot 2021-06-28 at 2 43 52 PM

Screenshot 2021-06-28 at 11 52 41 AM

### CODE REVIEW

File: website/src/pages/Blog/styles.ts [hunk 1]
```diff
@@ -45,17 +45,16 @@ const useStyles = makeStyles((theme: Theme) => ({
...
   },
   large: {
     width: '60px',
     height: '60px',
     marginRight: theme.spacing(1.8),
     [theme.breakpoints.down("xs")]: {
-      width: '32px',
-      height: '32px',
+      width: '48px',
+      height: '48px',
```
Comment (Pallavi-PH) [inline website/src/pages/Blog/styles.ts:57]: Why this change?

File: website/src/components/BlogsSlider/index.tsx [hunk 2]
```diff
@@ -45,7 +45,13 @@ const BlogsSlider: React.FC<BlogsSliderProps> = ({ recommendedBlogs }) => {
   }
 
   const getTags = (tags: Array<string>) => {
-    const tagItems = tags.map((tag) => <CustomTag blogLabel={tag} key={tag} />);
+    const tagItems = tags.map((tag) => {
+      return(
+        <div className={classes.tag}>
```
Comment (rakeshPRaghu) [inline website/src/components/BlogsSlider/index.tsx:50]: Add the key to this div

**Review by rakeshPRaghu — CHANGES_REQUESTED** [review]

File: website/src/pages/Blog/styles.ts (hunk 1 above)
Comment (Pallavi-PH) [reply website/src/pages/Blog/styles.ts:57]: It should be `32px` for mobile devices. https://www.figma.com/file/PkV24kmePqdjQiTBJMcJ1L/OpenEBS-landing-page-final?node-id=4%3A10250

Comment (IsAmrish) [reply website/src/pages/Blog/styles.ts:57]: ohh ok, I thought it's `32px` is by mistake. Because both `top author icon` as well as `author icon on the individual card ` was same.

File: website/src/components/BlogsSlider/style.ts [hunk 3]
```diff
@@ -14,10 +14,13 @@ const useStyles = makeStyles((theme: Theme) => ({
   },
   tagsWrapper: {
     display: 'flex',
-    gap: '12px',
     overflow: "scroll",
-    paddingBottom: '2px',
+    paddingBottom: '6px',
```
[the remark this answers is not in the document]
Comment (IsAmrish) [reply website/src/components/BlogsSlider/style.ts:18]: yes, many places it's been used mix of `px` vs `theme.spacing()`

**Review by rakeshPRaghu — APPROVED** [review]
LGTM

**Review by Pallavi-PH — APPROVED** [review]
lgtm. Thank you!

File: website/src/components/BlogsSlider/style.ts (hunk 3 above)
[the remark this answers is not in the document]
Comment (AVRahul) [reply website/src/components/BlogsSlider/style.ts]: for margin and padding, we are using the `theme.spacing()`, for any new changes we'll try to use the same, and if any other places `px` is been used,  we can update it gradually.

**Review by AVRahul — APPROVED** [review]

**Review by satyapriyamishra222 — APPROVED** [review]
